Website/src/rendering/src/assets/css/layout/information-section.css

```diff
@@ -21,7 +21,11 @@
       @apply md:col-span-3;
 
       .eyebrow {
-        @apply uppercase;
+        @apply font-medium hover:shadow-md uppercase bg-yellow inline-block mr-2;
```

Comment:
I would remove the hover style and the transition since this element is not clickable on the page. This gives a false signal that we can interact with this element.

Reply:
`mr-2` adds a margin right to the last element of that type. Instead it is recommended to add a margin-left to all elements but the first one with `.eyebrow + .eyebrow { @apply ml-2 }`